1-callbacks/solution.js

```diff
@@ -32,14 +32,45 @@ node solution.js name1 name2 name3
 ** give a look to node.js util.promisify, avoid to alter the validate-user.file **
 */
 
-function solution() {
-    // YOUR SOLUTION GOES HERE
 
-    // you get your 5 names here
 
+function solution() {
+    const namesForConsole = process.argv.slice(2);
+    const validateUser = require('./validate-user.js');
+    // you get your 5 names here
+    const usernames = ["Mary", "Julian", "Jose", "Stacy", "Gerardo"];
+    if (namesForConsole.length !== 0) {
+        for (let i = 0; i < namesForConsole.length; i++) {
+            const element = namesForConsole[i];
+            usernames.push(element)
+        }
+    }
     // iterate the names array and validate them with the method
+    setTimeout(() => {
+        console.log("\nsuccess")
+    }, 299)
+    setTimeout(() => {
+        console.log("failure")
+    }, 301)
```

Comment:
Could you give me more detail what is purpose of this piece of code? Why we need to wait for 299 milliseconds in the first setTimeout and 301 in the second one?
```
    setTimeout(() => {
        console.log("\nsuccess")
    }, 299)
    setTimeout(() => {
        console.log("failure")
    }, 301)
```